File: VideoIntelligence-Python-Service/src/schemas/file/api_urls_info_vo.py
from pydantic import BaseModel
from typing import Optional


# 定义接收参数的模型
class APIURLsInfoVO(BaseModel):
    title: Optional[str] = None  # 视频标题
    url: Optional[str] = None  # 视频链接
    cover: Optional[str] = None # 视频封面

File: vi-service/vi-service-file/vi-service-file-api/src/main/java/com/vi/entity/vo/APIURLsInfoVO.java
package com.vi.entity.vo;

import lombok.Data;

/**
 * 这是一个接收解析后的视频集合vo
 *
 * @author dotm
 */
@Data
public class APIURLsInfoVO {

    /**
     * 视频标题
     */
    private String title;

    /**
     * 视频链接
     */
    private String url;

    /**
     * 视频封面
     */
    private String cover;
}
